qa/L0_memory_growth/server_memory_mail.py

```diff
@@ -35,7 +35,11 @@
 
 if __name__ == '__main__':
     today = date.today().strftime("%Y-%m-%d")
-    subject = "Triton Server Memory Growth Summary: " + today
+    # Set the subject for weekly and nightly tests
+    if (sys.argv[1] == "weekly"):
```

Comment:
You can keep this, but for now we will just run this test weekly.

Reply:
Since the argument is now a string, you don't need the conditional, instead, you can just set the `subject` variable based on the argument `subject = "Triton Server Memory Growth " + sys.argv[1] + " Summary: " + today`

Reply:
Updated.